#!/usr/bin/env python
PACKAGE = "soma_ur5"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name                    Type      Reconfiguration level             Description  Default    Min   Max
gen.add("speed_gain", double_t, 0,   "Gain on speed",        0.1, 0.0, 2.0)

exit(gen.generate(PACKAGE, "soma_controller", "dyn_ur5_controller"))
